spec/requests/admin/watched_words_controller_spec.rb

```diff
@@ -26,6 +26,38 @@
     end
   end
 
+  describe '#create' do
+    context 'logged in as admin' do
+      before do
+        sign_in(admin)
+      end
+
+      it 'creates a word with default case-sensitivity' do
+        post '/admin/customize/watched_words.json', params: {
+          action_key: 'flag',
+          word: 'Deals'
+        }
+
+        expect(response.status).to eq(200)
+        expect(WatchedWord.take.case_sensitive?).to eq(WatchedWord.column_defaults['case_sensitive'])
```

Comment:
Since we're already testing elsewhere that `false` is the default value, there's no point in making this "expectation" somewhat harder to read/parse. Using the `false` literal is both easier and faster (no need to look up what's the default value for the column) to understand

```suggestion
        expect(WatchedWord.take.case_sensitive?).to eq(false)
```